notebook2/Makefile

```diff
@@ -36,7 +36,10 @@ $(IMAGE_SHORT_NAMES): %: build/images/%/_BUILT
 notebook-worker-images: $(PUSHED)
 	cat $^ > $@ && test $(APISERVER_JUPYTER_IMAGE) && echo $(APISERVER_JUPYTER_IMAGE) >> $@
 
-build: $(BUILT) notebook-worker-images
+hailjwt:
```

Comment:
this needs to be a `.PHONY` target or needs to depend on the contents of `../hailjwt`: `$(shell find ../hailjwt)`

Reply:
@akotlar